src/site/content/en/blog/better-responsiveness-metric/index.md

```diff
@@ -0,0 +1,508 @@
+      </tr>
+      <tr>
+        <td><code>mousedown</code></td>
+        <td><code>touchstart</code></td>
+      </tr>
+      <tr>
+        <td rowspan="5">Tap up / drag end</td>
+        <td><code>pointerup</code></td>
+        <td><code>pointerup</code></td>
+      </tr>
+      <tr>
+        <td><code>mouseup</code></td>
+        <td><code>touchend</code></td>
+      </tr>
+      <tr>
+        <td><code>click</code></td>
+        <td><code>mousedown</code></td>
+      </tr>
+      <tr>
+        <td></td>
+        <td><code>mouseup</code></td>
+      </tr>
+      <tr>
+        <td></td>
+        <td><code>click</code></td>
+      </tr>
+      <tr>
+        <td>Scroll</td>
+        <td colspan="3">N/A</td>
+      </tr>
+    </tbody>
+    <caption>DOM events for each interaction type.</caption>
+  </table>
+</div>
+
+The first three interactions listed above (keyboard, tap, and drag) are currently covered by FID.
+For our new responsiveness metric, we want to include scrolling as well, since scrolling is
+extremely common on the web and is a critical aspect of how responsive a page feels to users.
+
+{% Details %}
+
+{% DetailsSummary %}
+A note on start and end
+{% endDetailsSummary %}
+
+Note that each of these interactions has two parts: when the user presses the mouse, finger, or key
+down and when they lift it up. We need to ensure our metric doesn't count time the user spends
+holding the finger down between these two actions as part of the page's latency! {% endDetails %}
+
+##### Keyboard
+
+A keyboard interaction has two parts to it: when the user presses the key and when they release it.
+There are three associated events with this user interaction: `keydown`, `keyup`, and `keypress`.
+The following diagram illustrates the `keydown` and `keyup` delays and durations for a keyboard
+interaction:
+
+{% Img src="image/kns0INkO77RkiEStzHWYrugyWj32/zjRipPR9ioi551DUa3pj.jpeg", alt="Keyboard interaction
+with disjoint event durations", width="800", height="286" %}
+
+In the diagram above, the durations are disjoint because the frame from `keydown` updates is
+presented before the `keyup` occurs, but this does not need to be the case always. In addition, note
+that a frame can be presented in the middle of a task in the renderer process since the last steps
+required to produce the frame are done outside of the renderer process.
+
+The `keydown` and `keypress` occur when the user presses the key, while the `keyup` occurs when the
+user releases the key. Generally the main content update occurs when the key is pressed: text
+appears on the screen, or the modifier effect is applied. That said, we want to capture the more
+rare cases where `keyup` would also present interesting UI updates, so we want to look at the
+overall time taken.
+
+In order to capture the overall time taken by the keyboard interaction, we can compute the maximum
+of the duration of the `keydown` and the `keyup` events. The `keypress` event is deprecated, and it
```

Comment:
This detail about keypress feels like it should be in brackets, or some addendum?